tools/Azure.Mcp.Tools.StorageSync/tests/test-resources.bicep

```diff
@@ -0,0 +1,93 @@
+targetScope = 'resourceGroup'
+
```

Comment:
Introduce app to grant permissions to later in bicep.

@description('The client OID to grant access to test resources.')
param testApplicationOid string